lib/gollum/templates/navbar.mustache

```diff
@@ -41,21 +41,19 @@
       </div>
     </div>
 
-    {{#history}}
-      <div class="TableObject-item pl-2 hide-sm hide-md">
-        <a
-          class="btn btn-sm"
-          href="{{history_path}}/{{escaped_url_path}}"
-          id="minibutton-history"
-        >
-          Page History
-        </a>
-      </div>
-    {{/history}}
+    <div class="TableObject-item px-2">
+      <div class="BtnGroup" style="display: flex;">
```

Comment:
Is there a class for `display: flex` instead? Otherwise I think I'd prefer to have this styling in our custom CSS, rather than in the mustache template itself.

Reply:
Ah, it looks like there is a utility class: `d-flex`. 

I will update and rebase this PR to fix.